xmodule/capa/capa_problem.py

```diff
@@ -495,6 +503,41 @@ def get_grade_from_current_answers(self, student_answers):
 
         return newcmap
 
+    def get_grade_from_answers(self, student_answers, correct_map):
+        """
+        Gets the grade for the currently-saved problem state, but does not save it
+        to the block.
+
+        For new student_answers being graded, `student_answers` is a dict of all the
+        entries from request.POST, but with the first part of each key removed
+        (the string before the first "_").  Thus, for example,
+        input_ID123 -> ID123, and input_fromjs_ID123 -> fromjs_ID123.
+
+        For rescoring, `student_answers` is None.
+
+        Calls the Response for each question in this problem, to do the actual grading.
+        """
+        # old CorrectMap
```

Comment:
I think `old_cmap` is pretty self-explanatory. Should we remove these inline comments?